### ISSUE TITLE
Cannot load underlying module for 'AOModalStatus'

### ISSUE THREAD

**Issue by shadyk** [alecdoconnor/PuppyParadise#1]
Cannot load underlying module for 'AOModalStatus' for xcode 9

**Comment by alecdoconnor** [comment]
Can you provide more details?

**Comment by codeninja88** [comment]
It's not a huge deal, though as it shows the error but doesn't restrict the running of the app so it might simply be an Xcode thing. 
Basically, to see it, I simply downloaded the project as-is and it popped up with the error: "Cannot load underlying module for 'AOModalStatus"'" in the View Controller for PuppyParadise on line 10 (screenshot attached)
![screen shot 2018-04-20 at 10 44 43 am](https://user-images.githubusercontent.com/8423119/39025024-df05cbee-4487-11e8-8e79-103d072131a5.png)

FYI, i am running Xcode 9.2 on Sierra 10.12.6.

**Comment by alecdoconnor** [comment]
Please refer to this article, as it gives detailed instructions on getting started as well as properly linking libraries. Please let me know if you are still unable to fix this error, and update this issue if you are!
https://medium.com/flawless-app-stories/getting-started-with-reusable-frameworks-for-ios-development-f00d74827d11

**Comment by codeninja88** [comment]
I wasn't referring to you, Alec... i was referring to the original author of this thread as it has been a while since the thread was started. Hence i thought i'd provide more details in the author's place to let you know about the issue.

**Comment by alecdoconnor** [comment]
It sounds like an issue linking the two libraries. Try to unlink the projects, clean it (cmd-shift-K), and then relink them. Also make sure that it is building the framework when it builds the project. Let me know if that works for you!

**Comment by codeninja88** [comment]
Yep! That did the trick exactly as suggested. Thanks a bunch. :)

**Comment by alecdoconnor** [comment]
Perfect, glad to help! Shall I close out this issue?
